ga4gh/cli.py

```diff
@@ -816,3 +820,72 @@ def client_main(parser=None):
     else:
         runner = args.runner(args)
         runner.run()
+
+##############################################################################
+# Configuration testing
+##############################################################################
+
+
+class TestConfigExists(unittest.TestCase):
+    """This class implements configuration checks for the GA4GH server. It is
+    structured as a set of unit tests, and relies on the unittest module.
+    However, as some of the tests require state injection (through command
+    line arguments) we need to break the usual unittest pattern and instead
+    implement our own simple test runner.
+    """
+
+    def __init__(self, parser=None):
+        self.parser = parser
+
+    def setUp(self):
+        self.app = flask.Flask(__name__)
```

Comment:
I don't know if the configuration checker should ever start up a Flask instance; that seems outside the scope of what it's supposed to do...